kevm-pyk/src/tests/unit/test_kevm.py

```diff
@@ -50,3 +52,84 @@ def test_kinner_to_hex(test_id: str, input: KInner, result: KInner) -> None:
     to_hex = KEVM.kinner_to_hex(input)
     # Then
     assert to_hex == result
+
+
+JUMPDESTS_DATA: Final = [
+    ('empty', [], set_of([])),
+    (
+        'with_buf',
+        [
+            KToken(
+                'b"`\\xa0`@R4\\x80\\x15a\\x00\\x10W`\\x00\\x80\\xfd[P`@Qa\\x01\\x038\\x03\\x80a\\x01\\x03\\x839\\x81\\x01`@\\x81\\x90Ra\\x00/\\x91a\\x007V[`\\x80Ra\\x00PV[`\\x00` \\x82\\x84\\x03\\x12\\x15a\\x00IW`\\x00\\x80\\xfd[PQ\\x91\\x90PV[`\\x80Q`\\x9ba\\x00h`\\x009`\\x00`1\\x01R`\\x9b`\\x00\\xf3\\xfe`\\x80`@R4\\x80\\x15`\\x0fW`\\x00\\x80\\xfd[P`\\x046\\x10`(W`\\x005`\\xe0\\x1c\\x80c\\xa5m\\xfeJ\\x14`-W[`\\x00\\x80\\xfd[`S\\x7f\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x81V[`@Q\\x90\\x81R` \\x01`@Q\\x80\\x91\\x03\\x90\\xf3\\xfe\\xa2dipfsX\\"\\x12 \\xeb\\xb3\\x99\\x11\\xbe\\x13L\\xdeC\\xcc\\x01/\\x849\\xe7\\xc5\\x9aC\\xf1\\x0f\\xf4\\xdfE\\x14Z\\x80\\x90(\\xeb\\xda\\xee\\xeadsolcC\\x00\\x08\\r\\x003"',
+                'Bytes',
+            ),
+            KApply(
+                '#buf(_,_)_BUF-SYNTAX_Bytes_Int_Int',
+                [intToken(32), KVariable('VV0_x', 'Int')],
+            ),
+        ],
+        set_of(
+            [
+                intToken(16),
+                intToken(47),
+                intToken(55),
+                intToken(73),
+                intToken(80),
+                intToken(119),
+                intToken(144),
+                intToken(149),
+                intToken(187),
+            ]
+        ),
+    ),
+    (
+        'single_jumpdest',
+        [
+            KToken('b"["', 'Bytes'),
+        ],
+        set_of([intToken(0)]),
+    ),
+    (
+        'multiple_bytes',
+        [
+            KToken('b"\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00["', 'Bytes'),
```

Comment:
I think

```suggestion
            token(b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00['),
```

should work in cases like this.

Reply:
Indeed:

```python3
>>> from pyk.kast.inner import KToken
>>> from pyk.prelude.utils import token
>>> token(b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00[') == KToken('b"\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00["', 'Bytes')
True
```

Reply:
Nice! I didn't know `token` was a thing! I'll use it from now on instead.